data-types.md

```diff
@@ -3,4 +3,408 @@ title: Data Types
 toc: false
 ---
 
-Coming soon.
\ No newline at end of file
+CockroachDB supports the followinig data types. Click a type for more details.
+
+Type | Description | Example
+-----|-------------|--------
+[`INT`](#int) | A 64-bit signed integer. | `12345`
+[`DECIMAL`](#decimal) | An exact, fixed-point number. | `DECIMAL '1.2345'`
+[`FLOAT`](#float) | An inexact, floating-point number. | `1.2345`
+[`BOOL`](#bool) | A Boolean value. | `true` 
+[`DATE`](#date) | Year, month, day. | `DATE '2016-01-25'`
+[`TIMESTAMP`](#timestamp) | A date and time pairing. | `TIMESTAMP '2016-01-25 10:10:10'`
+[`INTERVAL`](#interval) | A span of time. | `INTERVAL '2h30m30s'`
+[`STRING`](#string) | A string of characters. | `'a1b2c3'`
+[`BYTES`](#bytes) | A string of binary characters. | `'a1b2c3'`
+
+## `INT`
+
+The `INT` type stores 64-bit signed integers, that is, whole numbers from 9,223,372,036,854,775,808 to 9,223,372,036,854,775,807. 
+
+#### Synonyms 
+
+In CockroachDB, the following are synonyms of `INT` and are implemented identically: 
+
+- `SMALLINT` 
+- `INTEGER` 
+- `INT64` 
+- `BIGINT`
+- `BITS`
```

Comment:
Hmm, I'd have expected `BIT` to be synonymous with `BOOL`, not `INT`. Postgres doesn't have a `BIT` type; mysql has one but it's parameterized (`BIT(n)`, which makes it more clear that it's a numeric type instead of a single bit). It looks like we support that too although we don't do anything with the parameter. We should document `BIT(n)` with the parameter instead of just saying that it's equivalent to `INT`.